src/main/docs/guide/introduction.adoc

```diff
@@ -1 +1,36 @@
-This Micronaut module specifies the Bill of Materials (BOM) for the Micronaut Project.
+This Micronaut module specifies the Bill of Materials (BOM) for the Micronaut Project, also known at the Micronaut Platform.
+
+The Micronaut Platform is a collection of modules that can be used to build Micronaut applications and have been verified to work well together.
+
+It aggregates many of the individual Micronaut module BOMs and provides a single BOM to manage the versions of all the modules.
+
+If you are using either the https://micronaut-projects.github.io/micronaut-gradle-plugin/latest/[Micronaut Gradle Plugin] or the https://micronaut-projects.github.io/micronaut-maven-plugin/latest/[Micronaut Maven Plugin], you do not need to add this module to your project as it will be added automatically.
```

Comment:
```suggestion
If you use either the https://micronaut-projects.github.io/micronaut-gradle-plugin/latest/[Micronaut Gradle Plugin] or the https://micronaut-projects.github.io/micronaut-maven-plugin/latest/[Micronaut Maven Plugin], you do not need to add this module to your project as it will be added automatically.
```